hardhat/test/KittyPartyComprehensive.test.ts

```diff
@@ -418,4 +418,30 @@ describe("KittyParty contracts comprehensive test", function() {
         parseFloat(ethers.utils.formatEther(kitten2Initial.toString()))
       ).to.be.equal(parseFloat(ethers.utils.formatEther(kitten2ReceiptBalance.toString())));
     });
+    
+    it('Cannot Stop staking without Completion of the given time.', async function() {
```

Comment:
Always use capital letter in the beginning of a sentence then don't mix caps, alternatively just use all small